mlflow/spark.py

```diff
@@ -418,12 +418,26 @@ def _should_use_mlflowdbfs(root_uri):
     # the Hadoop FileSystem API until a read call has been issued.
     from mlflow.utils._spark_utils import _get_active_spark_session
 
+    if (
+        not is_valid_dbfs_uri(root_uri)
+        or not is_databricks_acled_artifacts_uri(root_uri)
+        or not databricks_utils.is_in_databricks_runtime()
+        or environment_variables._DISABLE_MLFLOWDBFS.get() not in ["", "False", "false"]
```

Comment:
```suggestion
        or environment_variables._DISABLE_MLFLOWDBFS.get().lower() not in ["", "false"]
```

Reply:
```suggestion
        or environment_variables._DISABLE_MLFLOWDBFS.get().lower() == "true"
```

Is this more straightforward?

Reply:
It definitely is! Committed your second suggestion